interfaces/builtin/camera.go

```diff
@@ -34,11 +36,65 @@ const cameraConnectedPlugAppArmor = `
 /run/udev/data/c81:[0-9]* r, # video4linux (/dev/video*, etc)
 `
 
-// NewCameraInterface returns a new "camera" interface.
-func NewCameraInterface() interfaces.Interface {
-	return &commonInterface{
-		name: "camera",
-		connectedPlugAppArmor: cameraConnectedPlugAppArmor,
-		reservedForOS:         true,
+var cameraPermanentSlotUdev = []byte(`
+# RealSense UVC cameras (R200, F200, SR300 LR200, ZR300)
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a66", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0aa5", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0abf", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0acb", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0ad0", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="04b4", MODE:="0666", GROUP:="plugdev"
```

Comment:
@jdstrand I agree the group for realsense camera might be better to align with v4l conventional setup. About the acl mode, librealsense use libusb to claim realsense motion module usb interface https://github.com/IntelRealSense/librealsense/blob/v1.12.1/src/uvc-v4l2.cpp#L524  I've verified that acl mode 666 is still needed.
I'll update the pull request accordingly. Thanks your help!.